pkg/operator/ceph/cluster/osd/spec_test.go

```diff
@@ -331,6 +331,32 @@ func testPodDevices(t *testing.T, dataDir, deviceName string, allDevices bool) {
 	assert.Equal(t, 1, len(blkInitCont.VolumeDevices))
 	blkMetaInitCont = deployment.Spec.Template.Spec.InitContainers[8]
 	assert.Equal(t, 1, len(blkMetaInitCont.VolumeDevices))
+
+	// Test tune Fast settings when OSD on PVC
+	osdProp.tuneFastDeviceClass = true
+	deployment, err = c.makeDeployment(osdProp, osd, dataPathMap)
+	assert.NoError(t, err)
+	assert.Contains(t, deployment.Spec.Template.Spec.Containers[0].Args, "--osd-op-num-threads-per-shard-ssd=2")
```

Comment:
Loop over `defaultTuneFastSettings` instead of all those lines?

Reply:
Done.